tests/test_api_client.py

```diff
@@ -29,15 +30,15 @@ def test_create_simple_service():
     kind: Service
     apiVersion: v1
     metadata:
-      name: myservice
+      name: myservice-%s
     spec:
       selector:
         app: MyApp
       ports:
         - protocol: TCP
           port: 8089
           targetPort: 9369
-    """
+    """ % int(time.time() * 1000)
```

Comment:
We shouldn't be using this method of string formatting, please use f-strings instead. Check out the [coding guidelines](https://github.com/red-hat-storage/ocs-ci/blob/master/docs/coding_guidelines.md) for some examples.

Reply:
Hello. Thanks and please check now!